ush/python/pygfs/task/aero_analysis.py

```diff
@@ -0,0 +1,287 @@
+#!/usr/bin/env python3
+
+import os
+import glob
+import gzip
+import tarfile
+from netCDF4 import Dataset
+from logging import getLogger
+from typing import Dict, List, Any
+
+from pygw.attrdict import AttrDict
+from pygw.file_utils import FileHandler
+from pygw.timetools import to_isotime, to_fv3time, to_timedelta
+from pygw.fsutils import rm_p
+from pygw.template import Template, TemplateConstants
+from pygw.yaml_file import YAMLFile
+from pygw.logger import logit
+from pygfs.task.analysis import Analysis
+
+logger = getLogger(__name__.split('.')[-1])
+
+
+class AerosolAnalysis(Analysis):
+    """
+    Class for global aerosol analysis tasks
+    """
+    @logit(logger, name="AerosolAnalysis")
+    def __init__(self, config):
+        super().__init__(config)
```

Comment:
Since we're going to use the aersol test case as an example for the other tasks, I suggest puffing up the code with a few more inline comments. A short one here would probably help, something like `initialize pdy/cyc/...`